pkg/apperrors/errors.go

```diff
@@ -270,3 +270,28 @@ type InvalidEUAIDError struct {
 func (e *InvalidEUAIDError) Error() string {
 	return fmt.Sprint("EUAID ", e.EUAID, " does not correspond to a valid EUA user")
 }
+
+// InvalidStateError indicates that a provided enumerated state is not valid. Perhaps it is a string that has been cast as the enum type
+type InvalidStateError struct {
```

Comment:
If this is going to _just_ represent errors due to "enums" that have an invalid value, I think the name should be more specific, probably `InvalidEnumError`. It's not that important, though.

Reply:
You are absolutely right, thanks for catching that, I renamed the constructor method, but I forgot to rename the type.